### PR TITLE
Bug 1214280 - Users are unable to import contacts from memory card in…

### PR DESCRIPTION
**Pull request by ferjm**
… FTU/FTE. r=francisco

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cb79f9025f76099a00ea0b53a2dcbdbe70163af7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6ed317cb5afe5ba4e68653b519d1749bc38da9f1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1c21bdb538aa54c361ac4d725ab3f425549cb5f9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1a991630405ffecde8c2ec26dfd7e47381453661

File: apps/ftu/js/memcard_manager.js [hunk 1]
```diff
@@ -31,81 +35,103 @@ var SdManager = {
...
-          UIManager.navBar.removeAttribute('aria-disabled');
-          utils.overlay.hide();
-          if (!cancelled) {
+        var promises = [];
+        fileArray.forEach(file => {
+          promises.push(utils.sdcard.getTextFromFile(file, onContacts));
+        });
+
+        Promise.all(promises).then(() => {
+          window.setTimeout(() => {
```
Comment (arcturus) [inline apps/ftu/js/memcard_manager.js]: I don't think we will need this `window.setTimeout` since we have the promise that is already async

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=78736b3bf90a7d5ba075e7c87b101523bc2863af

**Pull request merged**